ts/Accessibility/ProxyProvider.ts

```diff
@@ -288,6 +288,24 @@ class ProxyProvider {
     public removeGroup(groupKey: string): void {
         const group = this.groups[groupKey];
         if (group) {
+            const { elements } = this.domElementProvider,
```

Comment:
The purpose of the `domElementProvider` is exactly to keep track of elements to be removed, so this functionality should probably be on a new `domElementProvider.removeElement` method, if we need to remove one element, but not all.

Reply:
Added `removeElement` method to `domElementProvider.removeElement` 👍